### PR TITLE
opencv 4.0.0

### PR DESCRIPTION
**Pull request by yuriy-yarosh**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

I'll update formulas for *caffe*, *siril*, *visp* - will change the `opencv` to `opencv@3` dependency.

### CODE REVIEW

**Comment by SMillerDev** [comment]
There's docs.brew.sh and a couple thousand examples in the repo. But if you're missing any documentation please make an issue for it or even better a pull request.

**Comment by SMillerDev** [comment]
There's docs.brew.sh and a couple thousand examples in the repo. But if you're missing any documentation please make an issue for it or even better a pull request.

**Comment by yuriy-yarosh** [comment]
For some weird reason I'm also getting this:
```
 brew audit siril --online
Error validating server certificate for 'https://free-astro.org:443':
 - The certificate is not issued by a trusted authority. Use the
   fingerprint to validate the certificate manually!
Certificate information:
 - Hostname: free-astro.org
 - Valid: from Sep 13 21:56:45 2018 GMT until Dec 12 21:56:45 2018 GMT
 - Issuer: Let's Encrypt Authority X3, Let's Encrypt, US
 - Fingerprint: B6:4E:04:66:E6:C0:4E:F5:06:C0:FC:C7:15:CC:6A:3C:77:A6:20:37
(R)eject, accept (t)emporarily or accept (p)ermanently?
```

File: Formula/opencv.rb [hunk 1]
```diff
@@ -1,14 +1,8 @@
 class Opencv < Formula
   desc "Open source computer vision library"
   homepage "https://opencv.org/"
-  url "https://github.com/opencv/opencv/archive/3.4.3.tar.gz"
-  sha256 "4eef85759d5450b183459ff216b4c0fa43e87a4f6aa92c8af649f89336f002ec"
-
-  bottle do
```
Comment (SMillerDev) [inline Formula/opencv.rb]: These should stay here.

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/opencv.rb (hunk 1 above)
Comment (yuriy-yarosh) [reply Formula/opencv.rb]: I've updated opencv to `4.0.0` and renamed this one to `opencv@3.rb`.
Also I've created a symlink in to Aliases dir `opencv@4` to `opencv.rb` with opencv4 formula, and removed the old symlink `opencv@3`, but for some reason it's still picking it.

**Comment by yuriy-yarosh** [comment]
Now, when I've faced all the weird parts of homebrew contribution, I do understand why folks tend to focus onto clumsy/buggy custom taps instead of contributing directly over here, - there's just too much hassle, and versioning is broken right away.

I don't think this community is viable enough, nowadays, to keep this project on going.

**Pull request closed without merging**